migrations/migrations.sql

```diff
@@ -378,3 +378,5 @@ alter table users add play_style int default 0 not null after preferred_mode;
 alter table users add custom_badge_name varchar(16) charset utf8 null after play_style;
 alter table users add custom_badge_icon varchar(64) null after custom_badge_name;
 alter table users add userpage_content varchar(2048) charset utf8 null after custom_badge_icon;
+
+# please refer to tools/migrate_v420.py for migrating to v4.2.0
```

Comment:
i should probably add the golang tool for this migration, the python one is stupidly slow on large datasets and pretty much not viable

Reply:
@tsunyoku working on go version

Reply:
available here: https://github.com/osuAkatsuki/gulag/tree/scores_table/tools/migrate_v420